src/components/ui/badge.tsx

```diff
@@ -27,16 +35,29 @@ const badgeVariants = cva(
 
 export interface BadgeProps
   extends React.HTMLAttributes<HTMLDivElement>,
-    VariantProps<typeof badgeVariants> {}
+    VariantProps<typeof badgeVariants> {
+  startIcon?: IconName;
+  endIcon?: IconName;
+}
```

Comment:
we might use radix icons or care icons. let's not do this. instead allow flexibility by passing it as children itself.

Reply:
Done